test/learning/linear_tests.jl

```diff
@@ -0,0 +1,125 @@
+using Unitful, UnitfulAtomic
+using InteratomicPotentials, InteratomicBasisPotentials
+using Random
+
+Random.seed!(0)
+
+ace = ACE(species = [:Na],         # species
+          body_order = 2,          # 4-body
+          polynomial_degree = 8,   # 8 degree polynomials
+          wL = 1.0,                # Defaults, See ACE.jl documentation 
+          csp = 1.0,               # Defaults, See ACE.jl documentation 
+          r0 = 1.0,                # minimum distance between atoms
+          rcutoff = 5.0)           # cutoff radius 
+
+# UnivariateLinearProblem ######################################################
+
+lb = LBasisPotentialExt(ace)
```

Comment:
Is "LBasisPotentialExt" the permanent new name, or is this just a placeholder before InteratomicBasisPotentialsExtension.jl is incorporated into the package?

Reply:
It is a temporary name and structure, unfortunately I have not been able to redefine the LBasisPotential structure inside PotentialLearning.jl to avoid the naming issue. After LBasisPotential is updated in InteratomicBasisPotentials.jl, this temporary structure can be removed from PotentialLearning.jl and we can return to the original name.